MQTTClient/MQTTFramework/MQTTFramework.h

```diff
@@ -1,30 +0,0 @@
-//
-//  MQTTFramework.h
-//  MQTT-Client-Framework
-//
-//  Created by Terry Latanville on 2015-11-25.
-//  Copyright © 2015-2017 Christoph Krey. All rights reserved.
-//
-
-#import <Foundation/Foundation.h>
-
-#import "MQTTSession.h"
-#import "MQTTDecoder.h"
-#import "MQTTSessionLegacy.h"
-#import "MQTTSessionSynchron.h"
-#import "MQTTMessage.h"
-#import "MQTTTransport.h"
-#import "MQTTCFSocketTransport.h"
-#import "MQTTCoreDataPersistence.h"
-#import "MQTTSSLSecurityPolicyTransport.h"
-#import "MQTTSessionManager.h"
-
-//! Project version number for MQTTFramework.
-FOUNDATION_EXPORT double MQTTFrameworkVersionNumber;
```

Comment:
Do we need these exports in `MQTTClient.h` now?

Reply:
@jcavar a good question. I'm not entirely sure which compnents consumes these. For the sake of concistency I could add them there